{"name": "Создание задачи с невалидной датой", "status": "passed", "steps": [{"name": "Отправляем POST запрос с невалидной датой: True", "status": "passed", "start": 1746535611787, "stop": 1746535612321}, {"name": "Проверяем код ответа и сообщение об ошибке", "status": "passed", "start": 1746535612321, "stop": 1746535612321}], "parameters": [{"name": "wrong_jsons", "value": "{'name': 'Test', 'due_date': True}"}], "start": 1746535611787, "stop": 1746535612321, "uuid": "d761367a-1441-448f-b607-9a2f17f0c8a6", "historyId": "e51203110afcbddf69a0111f8fde800a", "testCaseId": "cdf5097d6132b1714cb61dcaa05cd861", "fullName": "tests.api.test_tasks.TestTasks#test_create_task_invalid_date", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Egors-MacBook-Air.local"}, {"name": "thread", "value": "21580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}